### PR TITLE
Batch 1 (bad PR)

### LINKED ISSUE #49: Hunting and fixing Dead Links
**Describe the bug**

Right now the Documentation Site has a lot of dead links that redirect to nowhere. They mostly show the message `404 Not Found` which can create a bad impression for first-time contributors, looking to get started with the project.

**Expected behavior**

They should redirect to an appropriate page on the documentation site or to an appropriate resource. 

**Additional context**

Here are all the dead links: 

| Dead URL  | Linked Page |
| ------------- | ------------- |
| https://docs.moja.global/en/latest/DevelopmentSetup/git_and_github_guide  | https://docs.moja.global/en/latest/prerequisites/git.html  |
| https://docs.moja.global/en/latest/DevelopmentSetup/contributing/index  | https://docs.moja.global/en/latest/DevelopmentSetup/git_and_github_guide.html  |
| https://docs.moja.global/en/latest/GCBMDevelopmentSetup/prerequisites/python_3.7.html | https://docs.moja.global/en/latest/GCBMDevelopmentSetup/windows_installation.html |
| https://docs.moja.global/en/latest/GCBMDevelopmentSetup/prerequisites/microsoft_access_database_driver.html | https://docs.moja.global/en/latest/GCBMDevelopmentSetup/windows_installation.html | 
| https://docs.moja.global/en/latest/GCBMDevelopmentSetup/prerequisites/visual_c++_redistributable_packages.html | https://docs.moja.global/en/latest/GCBMDevelopmentSetup/windows_installation.html | 
| https://docs.moja.global/en/master/GCBMDevelopmentSetup/prerequisites/python_3.7.html | https://docs.moja.global/en/master/GCBMDevelopmentSetup/windows_installation.html |
| https://docs.moja.global/en/master/GCBMDevelopmentSetup/prerequisites/microsoft_access_database_driver.html | https://docs.moja.global/en/master/GCBMDevelopmentSetup/windows_installation.html |
| https://docs.moja.global/en/master/GCBMDevelopmentSetup/prerequisites/visual_c++_redistributable_packages.html | https://docs.moja.global/en/master/GCBMDevelopmentSetup/windows_installation.html |
| https://docs.moja.global/en/add_instructions_gdal/DevelopmentSetup/contributing/index | https://docs.moja.global/en/add_instructions_gdal/DevelopmentSetup/git_and_github_guide.html |
| https://docs.moja.global/en/add_instructions_gdal/DevelopmentSetup/git_and_github_guide | https://docs.moja.global/en/add_instructions_gdal/prerequisites/git.html |
| https://docs.moja.global/en/add_instructions_gdal/GCBMDevelopmentSetup/prerequisites/python_3.7.html | https://docs.moja.global/en/add_instructions_gdal/GCBMDevelopmentSetup/windows_installation.html | 
| https://docs.moja.global/en/add_instructions_gdal/GCBMDevelopmentSetup/prerequisites/microsoft_access_database_driver.html | https://docs.moja.global/en/add_instructions_gdal/GCBMDevelopmentSetup/windows_installation.html | 
| https://docs.moja.global/en/add_instructions_gdal/GCBMDevelopmentSetup/prerequisites/visual_c++_redistributable_packages.html | https://docs.moja.global/en/add_instructions_gdal/GCBMDevelopmentSetup/windows_installation.html | 
| https://docs.moja.global/en/master/DevelopmentSetup/git_and_github_guide | https://docs.moja.global/en/master/prerequisites/git.html |

We can follow the below strategy to solve this: 

- Take up a chunk of dead links and if we can find the apt page, add the link for the same. If not, then direct the link to some noteworthy resource as an alternative. 
- Contributors can take up batches of these dead link and raise PR for them easily, which would make it fast to remove all of them up.

### PR DESCRIPTION
**Pull request by chicken-biryani**
# Pull Request Template

## Description

It deals with the dead links on the page : 
[Windows Installation](https://docs.moja.global/en/master/GCBMDevelopmentSetup/windows_installation.html)
In all eliminating 3 dead links.
 
Fixes #49 

## Bug fix

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

--

## Additional Context (Please include any Screenshots/gifs if relevant)
The issue also deals with branches too, but the changes that have been made in this PR focus only on the main document.
...

## Checklist:

- [Yes] My code follows the style guidelines of this project
- [Yes] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [Yes] I have checked my code and corrected any misspellings
- [ ] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [Yes] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by chicken-biryani** [comment]
The issue took a deep dive and dealt with stale branches too. (Getting rid of them would be a good move maybe)
In all, there are actually just 5 dead links in the main.
I did the changes with 3 dead links, all of them had to lead to the same page, so I crammed them to a single link.
So that takes care of in all 9 dead links. ((2 branch + 1 main) * 3 dead link per)

File: docs/GCBMDevelopmentSetup/windows_installation.rst [hunk 1]
```diff
@@ -9,9 +9,11 @@ Before proceeding further, make sure you have the following prerequisites setup:
...
 Prerequisites
 -------------
-* `Python 3.7 <prerequisites/python_3.7.html>`_
-* `Microsoft Access Database Driver <prerequisites/microsoft_access_database_driver.html>`_
-* `Visual C++ Redistributable Packages <prerequisites/visual_c++_redistributable_packages.html>`_
+The prerequisites are:
+* Python 3.7
+* Microsoft Access Database Driver
+* Visual C++ Redistributable Packages
+You can view the details of the prerequisites `here. <https://docs.moja.global/en/master/GCBMDevelopmentSetup/prerequisites.html>`_
```
Comment (shubhamkarande13) [inline docs/GCBMDevelopmentSetup/windows_installation.rst:16]: @chicken-biryani 
Currently it looks like this:

> The prerequisites are: * Python 3.7 * Microsoft Access Database Driver * Visual C++ Redistributable Packages You can view the details of the prerequisites here.

Could you make it look like this?

>The prerequisites are: 
>
> - **Python 3.7**
> - **Microsoft Access Database Driver**  
> - **Visual C++ Redistributable Packages** 
> 
>Please set up the prerequisites by following the guide here.

Thanks! ✌

Comment (chicken-biryani) [reply docs/GCBMDevelopmentSetup/windows_installation.rst:16]: Apologies, my bad!

**Comment by chicken-biryani** [comment]
All the dead links, in issue number #49, have been taken care of in this pull request.

File: docs/prerequisites/git.rst [hunk 2]
```diff
@@ -12,4 +12,4 @@ To install Git, please refer to the official git installation instructions `here
 
 You can sign up for a Github account `here <https://github.com/join>`_.
 
-After successful installation of Git and GitHub account registration, you can proceed with further instructions on how to fork/clone the moja-global repositories `here <../DevelopmentSetup/git_and_github_guide>`_ .
+After successful installation of Git and GitHub account registration, you can proceed with further instructions on how to fork/clone the moja-global repositories `here <https://docs.moja.global/en/latest/DevelopmentSetup/git_and_github_guide.html>`_ .
```
Comment (shubhamkarande13) [inline docs/prerequisites/git.rst:15]: _After successful installation of Git and GitHub account registration ..._ to
_After successful installation of Git and registering your account on GitHub..._

Comment (chicken-biryani) [reply docs/prerequisites/git.rst:15]: Yes, sounds better. I'll do the changes!

**Pull request closed without merging**